spec/features/user_can_correct_submission_with_no_business_spec.rb

```diff
@@ -0,0 +1,111 @@
+require 'rails_helper'
+
+RSpec.feature 'Correcting a submission by reporting no business' do
```

Comment:
How about an additional spec that verifies the button isn't present when the feature env var is not present so we can be confident the feature-switch is working?